examples/data-objects/smde/package.json

```diff
@@ -89,15 +89,6 @@
 			}
 		}
 	},
-	"pnpm": {
-		"packageExtensions": {
-			"simplemde": {
-				"dependencies": {
-					"codemirror": "^5.65.11"
-				}
-			}
-		}
-	},
```

Comment:
This is much clearer than a root level override IMO. This is conceptually closer to what we want - we want to adjust codemirror's deps within this package. Overrides implies we want to use this version of the package everywhere. Yes, I know you scope the override, but even so this is closer to the semantic intent IMO.

Reply:
I agree would be a better way to do it. Sadly doesn't work since the constraints specified here were violated when I regenerated the lock file.